AppCenterDataStorage/AppCenterDataStorage/MSDataStore.h

```diff
@@ -35,11 +35,19 @@ static NSString *const MSDataStoreUserDocumentsPartition = @"user";
 static NSString *const MSDataStoreAppDocumentsPartition = @"readonly";
 
 /**
- * Time to live constants
+ * Cache does not expire.
  */
 static int const MSDataStoreTimeToLiveInfinite = -1;
+
+/**
+ * Do not cache.
```

Comment:
Why do we have TTL value for No Cache? It could have *never expire* but I don't think it makes sense to have *no cache*. Any use cases?

Reply:
@jaelim-ms PR #1500 now implements `read` API to check ReadOptions's TTL for NoCache and remove cache if needed. (see https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1500/files#diff-b33d4fad61841860c5d0921184064395R169)

Note that this PR is based on #1500.